### ISSUE TITLE
Carousel does not automatically start cycling on pageload

### ISSUE THREAD

**Issue by MohammedMubeen** [mihnsen/ui-carousel#48]
Hi Folks,

Please help me. Page load Carousel cycling automatic how to stop. after prev or next click only it should be cycling.

**Comment by concepta-thiago** [comment]
I have the same issue, was you able to find the solution?

**Comment by TarikTz** [comment]
When you load your content async it won't start autoplay. I'm trying to make some kind of a hack right now, I guess solution will be to simulate click on prev or next button.

**Comment by leap0ffaith** [comment]
Is there any update?

**Comment by nocerALE** [comment]
I have the same issue, @TarikTz did you find a solution ?

**Comment by TarikTz** [comment]
@nocerALE Yes I have, I've created timeout function which calls click triggerHandler on .carousel-next element. Here is a preview:

`   vm.handleAsyncLoad = () => {
        $timeout(function() {
          angular.element('.carousel-next').triggerHandler('click');
        }, 5000);
      };`

And I've called this functions after http get request return success.
